prot.c

```diff
@@ -435,7 +435,7 @@ process_queue()
 static job
 delay_q_peek()
 {
-    int i;
+    size_t i;
```

Comment:
@ysmolsky any possible issues with changing this type?

Reply:
As you can see "i" is used for accessing by index. It's increments from 0 and never goes into negative. size_t is exactly the type that should be used for that matter. Since we never write i to files we are okay to change it.

Reply:
And most important: `    for (i = 0; i < tubes.used; i++) {`

tubes.used is size_t, meaning that if ever tubes.used reaches value > max_int, then there would be an overflow in i.